### PR TITLE
fix: rowHeader checkbox click selection

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to the right branch according to our branching model
- [x] It's right to issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/helper/column.ts [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+
+  while (element) {
+    const name = element.getAttribute('data-column-name');
+    if (name && isRowHeader(name)) {
+      return true;
+    }
+    element = element.parentElement;
+  }
+  return false;
+}
```
Comment (seonim-ryu) [inline src/helper/column.ts]: dom.ts에 `findParent` 요건 못쓰려나요??
```js
findParent(target, 'cell-row-header')
```

Comment (seonim-ryu) [reply src/helper/column.ts]: https://github.com/nhn/tui.grid/pull/442

이거 머지되면 위에 함수 사용 가능합니다ㅎ

**Comment by js87zz** [comment]
리뷰 완료입니다!

**Comment by seonim-ryu** [comment]
[6/17] 리뷰 완료요ㅎ

**Pull request merged**